rust-axum/src/types/jwt.rs

```diff
@@ -31,10 +31,7 @@ pub struct JWTClaims {
 
 impl Display for JWTClaims {
   fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-    let expire = DateTime::<Utc>::from_utc(
```

Comment:
I'd keep the .ok_or(JWTError::Expired)? vs using expect. It is a prototype but I prefer to avoid using non total functions unless in a test function.

Reply:
Hi @darrell-roberts I have switched back to use the approach you suggested.